### PR TITLE
`showInlineError` is available in 'material' too

### PR DESCRIPTION
**Pull request by kubami**
Small docs update to reflect the code

### CODE REVIEW

**Comment by janowsiany** [comment]
I think that `showInlineError` is only affecting following fields: `DateField`, `LongTextField`, `NumField`, `SelectField`, `TextField` or am  i wrong? So please update this PR to match only those fields, also if you wish to add handling `showInlineError` in other type of fields then any contributions are welcome!

File: API.md [hunk 1]
```diff
@@ -227,6 +227,7 @@ import BoolField from 'uniforms-unstyled/BoolField'; // Choose your theme packag
     //   bootstrap3
     //   bootstrap4
     //   semantic
+    //   material
```
Comment (janowsiany) [inline API.md]: `BoolField` is not handling `showInlineError`

File: API.md [hunk 2]
```diff
@@ -477,6 +479,7 @@ import ListField from 'uniforms-unstyled/ListField'; // Choose your theme packag
...
```
Comment (janowsiany) [inline API.md]: The same with `ListField`

**Comment by kubami** [comment]
Of course, you are right!

**Pull request merged**